### DISCUSSION TITLE
Monthly schedule not behaving as expected

### DISCUSSION THREAD

**Discussion by enocom** [renovatebot/renovate#23179] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

We're updating the Cloud SQL Proxy repo to receive renovate updates monthly as part of our monthly release.

We have a config like so:

```
{
  "extends": [
    "config:base"
  ],
  "commitMessagePrefix": "deps: ",
  "postUpdateOptions": [
    "gomodTidy"
  ],
  "schedule": [
    "before 4am on the first day of the month"
  ]
}
```

From this config, I'd expect to see a bunch of renovate PRs on the first day of the month. However, renovate didn't open any PRs at the beginning of July. Am I doing something wrong here?

Thank you.

https://github.com/GoogleCloudPlatform/cloud-sql-proxy/blob/main/.github/renovate.json

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by nabeelsaabna** [comment]
Hi @enocom 
from what I can see in the job history there wasn't a run at all during the defined scheduled time:
```
  "schedule": [
    "before 4am on the first day of the month"
  ]
```

The first job that ran on July was at 2023-07-01 11:59:27am

**Reply by rarkins** [reply]
We'll try to look into why that happened. Normally repos should run every ~3 hours in the hosted app

**Reply by enocom** [reply]
Thank you. I appreciate it. It's good to know that our config is at least correct.

**Reply by enocom** [reply]
For what it's worth we made this configuration change on June 2. https://github.com/GoogleCloudPlatform/cloud-sql-proxy/pull/1828

**Reply by rarkins** [reply, marked as answer]
We've identified the root cause and will try to get it fixed before the start of next month. In short: our backend _also_ has some monthly work (checking every single disabled repo just in case) it schedules 30 minutes into 1st of the month and this ended up essentially blocking the hourly jobs for repos like this

**Reply by enocom** [reply]
That's great to hear. I assume changing to weekly would make this problem go away? Would you recommend any other adjustments to our schedule? Naturally a fix would be nice, but we can adjust in the meantime.

**Reply by rarkins** [reply]
I'd recommend you change to weekly at least temporarily so you get the PRs you missed. Alternatively use the Dependency Dashboard feature so you can request PRs on demand

**Reply by enocom** [reply]
Sounds good. Thanks again.
